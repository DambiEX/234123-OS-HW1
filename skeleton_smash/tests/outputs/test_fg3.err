smash error: job-id 1 does not exist
smash error: job-id 2 does not exist
smash error: fg: jobs list is empty
smash error: fg: jobs list is empty
