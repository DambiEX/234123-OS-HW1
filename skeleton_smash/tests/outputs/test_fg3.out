smash> smash> smash> smash> smash> smash> sleep 300& 5613
smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash: sending SIGKILL signal to 0 jobs:
